.circleci/config.yml

```
@@ -10,16 +10,16 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package-lock.json" }}
+            - v1-dependencies-{{ checksum "yarn.lock  " }}
```

I think this leading space is wrong.
```suggestion
            - v1-dependencies-{{ checksum "yarn.lock" }}
```